src/credentials/providers/sharedCredentialsProvider.ts

```diff
@@ -36,7 +36,7 @@ const SHARED_CREDENTIAL_PROPERTIES = {
  * Represents one profile from the AWS Shared Credentials files, and produces Credentials from this profile.
  */
 export class SharedCredentialsProvider implements CredentialsProvider {
-    private static readonly CREDENTIALS_TYPE = 'profile'
+    private static readonly CREDENTIALS_TYPE: CredentialSourceId = 'sharedCredentials'
```

Comment:
I believe this will affect what's displayed in the UI, e.g. by `updateCredentialsStatusBarItem()`.

Reply:
I am not sure this change makes sense. getCredentialsType() is supposed to return a credentials type, not a credentials source. Here are the definitions in `src/shared/telemetry/telemetry.gen.ts`:

```
export type CredentialSourceId = 'sharedCredentials' | 'sdkStore' | 'ec2' | 'envVars' | 'other'
export type CredentialType =
    | 'staticProfile'
    | 'staticSessionProfile'
    | 'credentialProcessProfile'
    | 'assumeRoleProfile'
    | 'assumeMfaRoleProfile'
    | 'ssoProfile'
    | 'ecsMetatdata'
    | 'ec2Metadata'
    | 'other'
```

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/1725